website/pages/docs/agent/config-entries/service-intentions.mdx

```diff
@@ -0,0 +1,330 @@
+---
+layout: docs
+page_title: 'Configuration Entry Kind: Service Intentions (beta)'
+sidebar_title: service-intentions <sup>Beta</sup>
+description: >-
+  The service-intentions config entry kind controls Connect traffic
+  authorization for both networking layer 4 (e.g. TCP) and networking layer 7
+  (e.g. HTTP).
+---
+
+# Service Intentions <sup>Beta</sup>
```

Comment:
```suggestion
# Service Intentions ((#service-intentions)) <sup>Beta</sup>
```
I didn't find anything which currently links to this. Updating it for consistency with other beta heading names (per these suggestions).